frontstage/views/passwords/forgot_password.py

```diff
@@ -43,17 +49,28 @@ def post_forgot_password():
             party_controller.reset_password_request(email)
         except ApiError as exc:
             if exc.status_code == 404:
-                logger.error('Requesting password change for email registered'
-                             ' on OAuth2 server but not in party service')
-                return render_template('passwords/forgot-password.check-email.html', email=email)
+                logger.error('Failed to retrieve details from party service')
+                return render_template('errors/404-error.html')
```

Comment:
I think this should be a 500 error, as if this happens it will be because something has gone wrong on our side

Reply:
i agree

Reply:
This exception handling has been removed altogether now as it's already being dealt with by party_controller.